apps/etterlatte-saksbehandling-ui/client/src/components/klage/vurdering/innhenting/KlageInfoInnhenting.tsx

```diff
@@ -0,0 +1,45 @@
+import { Klage } from '~shared/types/Klage'
+import { Content, ContentHeader, FlexRow } from '~shared/styled'
+import { HeadingWrapper } from '~components/behandling/soeknadsoversikt/styled'
+import { BodyLong, Button, Heading } from '@navikt/ds-react'
+import React from 'react'
+import { forrigeSteg, nesteSteg } from '~components/klage/stegmeny/KlageStegmeny'
+import { useNavigate } from 'react-router-dom'
+
+export function KlageInfoInnhenting(props: { klage: Klage }) {
+  const { klage } = props
+
+  const navigate = useNavigate()
+  return (
+    <Content>
+      <ContentHeader>
+        <HeadingWrapper>
+          <Heading level="1" size="large">
+            Hent inn informasjon fra søker
+          </Heading>
+        </HeadingWrapper>
+        <BodyLong spacing={true}>
+          Hent inn mer informasjon fra søker for å avklare om formkravene kan innfris.
+          <br />
+          Du kan lage nytt brev til søker fra brevsiden, og oppgaven kan inntil videre settes på vent ved å utvide
+          fristen.
+        </BodyLong>
+
+        <FlexRow justify="left">
+          <Button type="button" variant="tertiary" onClick={() => navigate(`/person/${klage.sak.ident}?fane=BREV`)}>
```

Comment:
```suggestion
          <Button as="a" variant="tertiary" href={`/person/${klage.sak.ident}?fane=BREV`>
```
Bedre å sette lenke-attributt og ha html-innhold som en anchor-tag med tanke på UU / semantikk